app.py

```diff
@@ -20,5 +21,28 @@ def Login():
             flash('Login Unsuccessful. Invalid Email/Password')
     return render_template('login.html',title='Login | SAC Portal, IIT Mandi',form=form)
 
+#Example  https://host/leaveclub?userID='B19188'&clubID='C10001'
+@app.route('/leaveclub', methods = ['POST'])
+def leave_club():
+    data = {
+        "tablename" : "ClubMembers",
+        "userID" : request.args.get('userID'),
```

Comment:
use request body to transfer data, instead of query parameters,
this'll allow for direct HTML form calls

Reply:
yes, userstood for sure, will take care from next time in PRs
However, changing it and doing it as vipul suggested, to not take any parameter, isnted use the session value.